rust/src/lib/net_state.rs

```diff
@@ -1,8 +1,5 @@
 // SPDX-License-Identifier: Apache-2.0
 
-#[cfg(not(feature = "gen_conf"))]
-use std::collections::HashMap;
```

Comment:
This should not be removed ? 

`rust_lint` nightly CI is failed with the error:
```
error[E0412]: cannot find type `HashMap` in this scope
   --> src/lib/net_state.rs:267:17
    |
267 |     ) -> Result<HashMap<String, Vec<(String, String)>>, NmstateError> {
    |                 ^^^^^^^ not found in this scope
    |
help: consider importing this struct
    |
3   + use std::collections::HashMap;
```

Reply:
+1